addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayBridgeHandler.java

```diff
@@ -107,6 +110,8 @@ public void run() {
                 if (mZWayApi.getLogin() != null) {
                     // Thing status set to online in login callback
                     logger.info("Z-Way bridge successfully authenticated");
+                    // Gets the latest deviceList from zWay during bridge initialization
+                    deviceList = mZWayApi.getDevices();
```

Comment:
Yes, thats the right way to do it! Nice.

Reply:
Perfect. Thanks. :)